src/App.js

```diff
@@ -87,6 +89,26 @@ export default class App extends Component {
     }
   }
 
+  /* * * * * * * * * * * * * * * * *
+   *
+   * SET KONAMI EASTER EGG
+   *
+   * * * * * * * * * * * * * * * * */
+  setKonami () {
+    if (this.props.meta.konami === '') return ;
+    const konami = '38,38,40,40,37,39,37,39,66,65'
+    const keys = []
+    const listener = (e) => {
+    	keys.push(e.keyCode)
+    	if (keys.join(',').includes(konami)) {
+        console.log("test")
+    		document.removeEventListener('keydown', listener)
+    		window.location.href = this.props.meta.konami
```

Comment:
Tu arrêtes d'écouter une fois que le code est tapé une première fois ?
Pas besoin de décider ici de l'action à mener, juste un petit this.setState({ konami_mode: true })